frontend/src/types/api-platform.ts

```diff
@@ -7,51 +7,12 @@
  * file that was distributed with this source code.
  */
 
-import { User } from '@/types/user';
 import { Fishbowl as apiPlatformFishbowl } from './api-platform/interfaces/fishbowl';
-import { faker } from '@faker-js/faker';
 
-export interface Fishbowl extends Omit<apiPlatformFishbowl, 'startDateTimeTz' | 'endDateTimeTz'> {
+export interface Fishbowl
```

Comment:
Here we override the api platform definitions. Previously we were already doing the same for startDateTimeTz and endDateTimeTz. I have added slug and id.

Reply:
:eyes:

Reply:
👍🏽